### DISCUSSION TITLE
Smoother zoom

### DISCUSSION THREAD

**Discussion by ArystanK** [mipt-npm/maps-kt#3] [Ideas]
Currently, zoom works discretely. Mb we could make it so it would seem smoother. For reference, we could see implementations in the android MapCompose library or multiplatform compose-MapView library, which both have smooth zooming.
MapCompose library - https://github.com/p-lr/MapCompose
compose-MapView - https://github.com/avdim/compose-mapview

**Comment by altavir** [comment]
PRs would be welcome. The change seems to be cosmetic, but nice to have. I will make an issue about it.
